psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -310,18 +317,3 @@ def compute_hessian(self, molecule: 'psi4.core.Molecule',
         molclone.fix_orientation(True)
         molclone.fix_com(True)
 
-        # Record undisplaced symmetry for projection of diplaced point groups
-        core.set_global_option("PARENT_SYMMETRY", molecule.schoenflies_symbol())
```

Comment:
this is newer and good -- assoc. with mostly killing off a global, iirc. https://github.com/psi4/psi4/blame/master/psi4/driver/procrouting/empirical_dispersion.py#L314